File: Lab02/combinador.c
#include <stdio.h>
#include <string.h>

int main(){

    char str1[100];
    char str2[100];

    char str3[200];
    
    int t = 0;

    if(strlen(str2) < strlen(str1) ){
        t = strlen(str1);
    }
    else{
        t = strlen(str2);
    }
    int x = 0;
    int j = 0;
    for(int i = 0; i < t; i+=x ){
        if(j > strlen(str1)){
                str3[i] = str2[j];
                x = 1;
        }else if(j > strlen(str2)){
            str3[i] = str1[j];
            x = 1;
        }
        else{
            str3[i] = str1[j];
            str3[i+1] = str2[j];
            x = 2;
        }
        j++;
    }
    
        printf("%s", str3);

    return 0;
}

File: Lab02/combinador.java
class Combinador{

public static Strign programa(String str1, String str2){
	int x, y;
	boolean primeiro = false; 
	if(str1.length() > str2.length()){
		x = str1.length();
		y = str2.length();
		primeiro = true;
	}else{
		y = str1.length();
		x = str2.length();
	}
	String resultado = "";
	for(int i = 0; i < x; i++){
		if(primeiro == true && i+1 >= y){
		resultado = resultado + str1.charAt(i);
		}else if(primeiro == false && i+1 >= y){
		resultado = resultado + str2.charAt(i);
		}else{
		resultado = resultado + str1.charAt(i) + str2.charAt(i);
		}
	}
}




public static void main(String[] arg){
String str1 = MyIO.readLine();
String str2 = MyIO.readLine();

while(){

}

}
}
